desktop/windows/install.md

```diff
@@ -102,14 +102,18 @@ Looking for information on using Windows containers?
 
 ## Install Docker Desktop on Windows
 
+### Install interactively
+
 1. Double-click **Docker Desktop Installer.exe** to run the installer.
 
     If you haven't already downloaded the installer (`Docker Desktop Installer.exe`), you can get it from
     [**Docker Hub**](https://hub.docker.com/editions/community/docker-ce-desktop-windows/).
     It typically downloads to your `Downloads` folder, or you can run it from
     the recent downloads bar at the bottom of your web browser.
 
-2. When prompted, ensure the **Enable Hyper-V Windows Features** or the **Install required Windows components for WSL 2** option is selected on the Configuration page.
+2. When prompted, ensure the **Use WSL 2 instead of Hyper-V** option on the Configuration page is selected or not depending on your choice of backend.
```

Comment:
```suggestion
2. When prompted, ensure you've selected the right backend on the Configuration page. Select **Use WSL 2 instead of Hyper-V** only if your system supports this option.
```

Reply:
If the system doesn’t support one of the options, it will not be available to the user.